Marlin/src/module/probe.cpp

```diff
@@ -479,6 +479,26 @@ bool Probe::probe_down_to_z(const float z, const feedRate_t fr_mm_s) {
   return !probe_triggered;
 }
 
+#if ENABLED(PROBE_NEEDS_TARE)
+bool Probe::tare_z_probe() {
+  #if ENABLED(PROBE_ENABLE_WINDOW)
+    if ((READ(PROBE_ENABLE_PIN) == PROBE_NEEDS_ENABLE_STATE)) {
```

Comment:
When leveling and probing the *x* points on the bed I think the X-axis doesn't raise high enough to make PROBE_ENABLE_PIN `LOW` - so I think we need to get rid of this check.

Reply:
Id go the other way and force the raise between probes to be high enough to clear this. If we power up with the nozzle on the bed and allow a tare there, or have the nozzle stuck to a part, or any other odd failure condition we really want to see this pin drop when we tare.

Reply:
That would make the leveling procedure a good bit longer.

Reply:
Quick look at the machine here, raise height is still less than typical BLTouch deploy clearance so IMO the additional raise height is minimal compared to the additional safety constraint gained. However that can be very easily disabled and the switch ignored during tare by flipping the toggle off if the user wants to cut it close.

Reply:
It would make it a bit longer, thats why theres an explicit toggle on the functionality. I heavily advise users to probe once carefully, store the mesh, and recall until something changes. If they probe every time, they can either take the time and safety, or turn off the toggle. Originally all but one call, the second slow probe, ignored the check. That means if it was going to crash, it already would have by the time the only call using it would get run so it should function as it was with the check off.

Reply:
Discussed more on discord and intentions are understood